unlock-app/src/components/creator/CreatorLocks.js

```diff
@@ -28,9 +28,14 @@ export class CreatorLocks extends React.Component {
   }
 
   render() {
-    const { locks, createLock } = this.props
+    const { locks, transactions, createLock } = this.props
     const { showDashboardForm } = this.state
-    let lockFeed = Object.values(locks).reverse() // We want to display newer locks first
+
+    // We want to display newer locks first, so sort the locks by blockNumber in descending order
+    const comparator = (a, b) =>
```

Comment:
can you do this in `mapStateToProps` for the dashboard instead? I.e. generate lockFeed there, it will keep the logic out of the component.

Reply:
Yeah, definitely

Reply:
Hm, hoisting it out of CreatorLocks causes some weirdness. Working on it, but might take a while.